docs/installation/autoware/docker-installation.md

```diff
@@ -24,7 +24,7 @@
 ### Installing dependencies manually
 
 - [Install Docker Engine](https://github.com/autowarefoundation/autoware/tree/main/ansible/roles/docker_engine#manual-installation)
-- [Install Docker Compose](https://github.com/autowarefoundation/autoware/tree/main/ansible/roles/docker_compose#manual-installation)
+- [Install Docker Compose](https://github.com/autowarefoundation/autoware/tree/main/ansible/roles/docker_engine#manual-installation)
```

Comment:
This change seems completely unrelated to the other changes for the PR? If this is just fixing the link, then it should be submitted as a separate PR, as a simple fix PR would get approved almost immediately rather than getting stuck in a more complex PR like this one that is dependent on other PRs getting merged first and a potentially long review process.

Reply:
I've fixed this in https://github.com/autowarefoundation/autoware-documentation/pull/133.